ERCS/erc-7595.md

```diff
@@ -0,0 +1,491 @@
+---
+eip: 7595
+title: Collateralized NFT
+description: ERC-721 Extension to enable collateralization with ERC-20 based tokens.
+author: 571nKY (@571nKY), Cosmos (@Cosmos4k), f4t50 (@f4t50), Harpocrates (@harpocrates555)
+discussions-to: https://ethereum-magicians.org/t/collateralized-nft-standard/18097
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-13
+requires: 20, 721
+---
+
+## Abstract
+
+This proposal recommends an extension of [ERC-721](./eip-721.md) to allow for collateralization using a list of [ERC-20](./eip-20.md) based tokens. The proprietor of this ERC collection could hold both the native coin and [ERC-20](./eip-20.md) based tokens, with the tokenId acting as the access key to unlock the associated portion of the underlying [ERC-20](./eip-20.md) balance.
```

Comment:
This is likely a me problem, but I'm not sure your abstract gives enough detail. Could you maybe add a sentence or two that describes the intended flow? Keep it brief, and try not to repeat your other sections.

---

What I'm getting from reading _just_ the abstract is like... anyone can deposit ether or ERC-20 tokens into the 721 contract (associated with a particular tokenId), and the owner of that token can withdraw the ether/ERC-20? Why?

Reply:
Added to the commit